pwndbg/commands/telescope.py

```diff
@@ -39,7 +39,7 @@ def telescope(address=None, count=telescope_lines, to_string=False):
     ($sp by default)
     """
     address = int(address if address else pwndbg.regs.sp) & pwndbg.arch.ptrmask
-    count   = int(count) & pwndbg.arch.ptrmask
+    count   = max(int(count), 0) & pwndbg.arch.ptrmask
```

Comment:
i got the idea what its supposed to do, removing the block and i will rework this part in a new PR